python/pyxbos/pyxbos/drivers/dark_sky/dark_sky.py

```diff
@@ -1,118 +0,0 @@
-from pyxbos.driver import *
```

Comment:
Why is this file deleted? Current installations are using this driver. Please undelete or rename your other driver file to this

Reply:
My bad! Will revert this deletion. 

The new driver that I added has a lot more fields including a calculation solar irradiance measurement.

Reply:
If the new driver is fully backwards compatible, then why not replace the `dark_sky.py` driver with the new driver but keep the same file name?

Reply:
It actually isn't. To calculate the solar irradiance, you need to install a new python library. I think it'll be better if we keep the two files separate. 

Additionally, in the new driver both the predictions and the current values are part of the same class - to avoid exhausting the "free" API calls using the API key.